### PR TITLE
opae-sdk: remove ase rpm from spec file

### PR DESCRIPTION
**Pull request by anandaravuri**
Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: opae.spec.fedora [hunk 1]
```diff
@@ -16,7 +16,7 @@ Requires:  systemd
 Requires:  tbb
 Requires:  hwloc
 Requires:  spdlog
-Requires:  libedit
+Requires:  edit
```
Comment (tswhison) [inline opae.spec.fedora:19]: We want libedit.  Remember from before?

**Review by tswhison — DISMISSED** [review]
You could have simply done a
$ git revert
on the commit where I added these changes.  It would have been cleaner, and would have removed the issue of injecting new changes that are unrelated, eg the libedit change.

File: opae.spec.fedora (hunk 1 above)
Comment (anandaravuri) [reply opae.spec.fedora:19]: rpmlint error opae.x86_64: E: explicit-lib-dependency libedit
changed libedit -> edit

Comment (tswhison) [reply opae.spec.fedora:19]: package libedit and edit are different.  We discussed this before.  Please change it back to libedit.

Comment (anandaravuri) [reply opae.spec.fedora]: changed  back to libedit.

**Review by tswhison — APPROVED** [review]

**Pull request merged**